capstone/src/main/java/com/google/step/servlets/RestaurantsServlet.java

```diff
@@ -37,7 +37,19 @@
 public class RestaurantsServlet extends HttpServlet {
   private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   /**
-   * Returns a list of Restaurants.
+   * Returns a list of RestaurantHeaders, with a snapshot of restaurant details.
+   * @param response A list of restaurant details, in the following json format:
+    {
+      "restaurants" : [
```

Comment:
Nit: Since this is a list of restaurant headers, consider the property name "restaurantHeaders"